server/src/main/java/org/opensearch/extensions/ExtensionsOrchestrator.java

```diff
@@ -71,6 +74,7 @@ public class ExtensionsOrchestrator implements ReportingService<PluginsAndModule
     public static final String REQUEST_EXTENSION_LOCAL_NODE = "internal:discovery/localnode";
     public static final String REQUEST_EXTENSION_CLUSTER_SETTINGS = "internal:discovery/clustersettings";
     public static final String REQUEST_EXTENSION_REGISTER_REST_ACTIONS = "internal:discovery/registerrestactions";
+    public static final String REQUEST_EXTENSION_REGISTER_SETTINGS = "internal:discovery/registersettings";
```

Comment:
Can we have the action in the format `internal:extensions...`

Reply:
This is an open question in #4334.  For now I'd like to leave it the same as the others.